django/db/models/query.py

```diff
@@ -635,6 +635,9 @@ def update(self, **kwargs):
         self._for_write = True
         query = self.query.clone(sql.UpdateQuery)
         query.add_update_values(kwargs)
+        # Clear any annotations so that they won't be present in subqueries
+        query.set_annotation_mask(None)
```

Comment:
I guess I'd rather omit it so that if it is needed, we get a report about it and can add a test.